input/pagecontent/3-6-3-diagnosticreport-update.md

```diff
@@ -2,16 +2,16 @@
 
 eventMaturity | [1 - Submitted](3-1-2-eventmaturitymodel.html)
 
- The `DiagnosticReport-update` event is used by clients to support content sharing in communication with a Hub which also supports content sharing.  A `DiagnosticReport-update` request will be posted to the Hub when an application desires a change be made to the current state of exchanged information or to add or remove a reference to a FHIR resource contained in the content of the current anchor context. One or more updates MAY occur while the anchor context is open.
+ The `DiagnosticReport-update` event is used by clients to support content sharing in communication with a Hub which also supports [content sharing](4-6-content-sharing.html).  A `DiagnosticReport-update` request will be posted to the Hub when an application desires a change be made to the current state of exchanged information or to add or remove a reference to a FHIR resource contained in the content of the current DiagnosticReport. One or more updates MAY occur while the anchor context is open.
```

Comment:
Generally, it's unclear to me that replacing the more concrete term DiagnosticReport with the less specific "anchor context" is an improvement.

Reply:
Then we should not make that change ;)

Reply:
Discussed during InM FHIRcast call:
* in all scenarios, for the DxReport-update event, DxReport is the anchor context. 
Logged a jira to come back to re-evaluate the use of the term on this page. Closing this conversation for now.